pyserini/vectorizer.py

```diff
@@ -36,23 +36,21 @@ class TfidfVectorizer:
     """
 
     def __init__(self, lucene_index_path: str, min_df: int = 1, verbose: bool = False) -> None:
```

Comment:
I suppose `min_df` is still applicable for filtering the vocab, even if we're ultimately computing BM25. Is that the intention?

Reply:
Yes, with BM25Vectorizer we can still use `min_df` to filter out rare words when building our vocabulary.